test/sum.sh

```diff
@@ -36,6 +36,32 @@ if [ $# -gt 0 ]; then
     exit -1
 fi
 
+if [ -z "$(which clixon_util_xml)" ]; then
```

Comment:
This probably should be done in lib.sh
The reason is that "sum.sh" is just one way to run the tests, there are others, such as individually, ./mem.sh, ./all.sh

Reply:
Dropped.